test/component/deceased/testAddress.js

```diff
@@ -1,19 +1,21 @@
 'use strict';
 
 const TestWrapper = require('test/util/TestWrapper');
-const DocumentUpload = require('app/steps/ui/documentupload');
+const DiedEnglandOrWales = require('app/steps/ui/deceased/diedengorwales');
 const testCommonContent = require('test/component/common/testCommonContent.js');
+const probateJourney = require('app/journeys/probate');
 
 describe('deceased-address', () => {
+    const ftValue = {ft_new_deathcert_flow: true};
     let testWrapper;
```

Comment:
Is this ok ? It seems we're adding feature testing here rather than removing

Reply:
Good spot! Have fixed this